tenants.html

```diff
@@ -1,162 +1,153 @@
-              с 09:00 до 19:00
-              <span id="text-open" class="text-success d-none">
-                 (Сейчас открыто)
-              </span>
-            </li>
-          </ul>
-        </div>
-      </div>
+  <div class="header__menuwrapper header__menuwrapper__top d-none" id="menu">
+    <div class="container">
+      <ul class="header__menu">
+        <li><a href="index.html">Главная</a></li>
+        <li><a href="visitors.html">Посетителям</a></li>
+        <li><a href="tenants.html" class="list-group-item disabled">Арендаторам</a></li>
+        <li><a href="owners.html">Собственникам</a></li>
+      </ul>
+    </div>
+  </div>
 
+  <div class="top-div">
     <div class="container pb-3">
       <ul class="list-unstyled">
-        <li>Помещения, доступные для сдачи в аренду</li>
-        <li>Информация об управляющей компании</li>
-        <li>Возможность разместить объявление об акциях и распродажах</li>
+        <li>
+          <i class="fas fa-map-marker-alt text-secondary"></i>
+          <a
+            href="https://yandex.ru/maps/org/dve_pyatyorki/1024420856/?ll=20.509270%2C54.727694&oid=1024420856&ol=biz&z=18">
+            <span>
+              Калининград, улица Горького, 55
+            </span>
+          </a>
+        </li>
+        <li>
+          <span>
+            <i class="far fa-clock text-secondary"></i>
+          </span>
+          с 09:00 до 19:00
+          <span id="text-open" class="text-success d-none">
+            (Сейчас открыто)
+          </span>
+        </li>
       </ul>
     </div>
-    <footer class="container pb-5">
+  </div>
 
-      <p>
-        <!-- Yandex.Metrika informer -->
-        <a
-          href="https://metrika.yandex.ru/stat/?id=53046907&amp;from=informer"
-          target="_blank"
-          rel="nofollow"
-          ><img
-            src="https://informer.yandex.ru/informer/53046907/3_1_FFFFFFFF_FFFFFFFF_0_pageviews"
-            style="width:88px; height:31px; border:0;"
-            alt="Яндекс.Метрика"
-            title="Яндекс.Метрика: данные за сегодня (просмотры, визиты и уникальные посетители)"
-            class="ym-advanced-informer"
-            data-cid="53046907"
-            data-lang="ru"
-        /></a>
-        <!-- /Yandex.Metrika informer -->
-      </p>
-    </footer>
+  <div class="container pb-3">
+    <ul class="list-unstyled">
+      <li>Помещения, доступные для сдачи в аренду</li>
+      <li>Информация об управляющей компании</li>
+      <li>Возможность разместить объявление об акциях и распродажах</li>
+    </ul>
+  </div>
 
-    <script type="text/javascript">
-      // https://stackoverflow.com/a/800010/1775459
-      document.addEventListener("DOMContentLoaded", function() {
-        var element = document.getElementById("text-open");
-        if (element) {
-          var kaliningradHours = new Date().getUTCHours() + 2;
-          var isMallOpened = kaliningradHours >= 9 && kaliningradHours < 19;
+  <div class="container pt-3 pb-3 text-center">
```

Comment:
Seems like everything okey however I don't get why you diceded to use direct link to index.html instead of '/' ? 
It would be good to use a related link in case the name of index page would be changed and we don't need to change href in many places.

Reply:
If the website is uploaded in a subfolder, relative "/" link will point to its parent. That's why I chose "index.html" to be the link. However, seems to me that this is not the case here, and the site won't be uploaded in a subfolder so i'll chance the links to the "/".

Reply:
It shoudn't impact on deployed version.